addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/timing/OnlineSimpleLinearRegression.java

```diff
@@ -17,7 +17,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.zaproxy.zap.extension.ascanrules.timing;
+package org.zaproxy.addon.commonlib.timing;
 
 /**
```

Comment:
Add the `@since` JavaDoc tag to the moved non-test classes, e.g. `@since 1.15.0`.